lib/postgrex/extensions/tsvector.ex

```diff
@@ -0,0 +1,45 @@
+defmodule Postgrex.Extensions.Tsvector do
+  @moduledoc false
+  @behaviour Postgrex.Extension
+
+  def init(_), do: {}
+
+  def matching(_), do: [send: "tsvectorsend"]
```

Comment:
Alright, I'll have a look at the C code of postgresql and implement the binary parsing of TSVector.

I'll poke you once ready as it may take some reading. afaik, the decoding part requires a small algorithm.